### PR TITLE
use Stripe js object after script load

### PR DESCRIPTION
**Pull request by serdyuka**

### CODE REVIEW

**Comment by serdyuka** [comment]
@makasim ready for review

File: src/Payum/Stripe/Resources/views/Action/obtain_js_token.html.twig [hunk 1]
```diff
@@ -45,26 +45,6 @@
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```
Comment (makasim) [inline src/Payum/Stripe/Resources/views/Action/obtain_js_token.html.twig]: could you create a separate block payum_vendor_javascripts and move these to it.

File: src/Payum/Stripe/Resources/views/Action/obtain_js_token.html.twig [hunk 2]
```diff
@@ -72,11 +52,31 @@
                 // Disable the submit button to prevent repeated clicks
                 $form.find('button').prop('disabled', true);
 
+                // This identifies your website in the createToken call below
+                Stripe.setPublishableKey({{ publishable_key|json_encode|raw }});
```
Comment (makasim) [inline src/Payum/Stripe/Resources/views/Action/obtain_js_token.html.twig]: not sure it must be in submit.

**Comment by makasim** [comment]
@serdyuka ping

**Comment by serdyuka** [comment]
@makasim done

**Comment by serdyuka** [comment]
@makasim done

**Comment by makasim** [comment]
Thanks.

**Pull request merged**